docs/parameter_files.rst

```diff
@@ -139,15 +139,15 @@ Telescope Configuration
 
     .. literalinclude:: example_configs/bl_lite_mixed.yaml
 
-    This yaml file provides the telescope name, location in latitude/longitude/altitude in degrees/degrees/meters (respectively), and the `beam dictionary`. In this case, beam_id == 0 is the UVBeam file written out to hera.uvbeam, and beam_id == 1 is an Airy beam with diameter 16m. The dictionary only needs to be as long as the number of unique beams used in the array, while the layout file specifies which antennas will use which beam type. This allows for a mixture of beams to be used, as in this example.
+    This yaml file provides the telescope name, location in latitude/longitude/altitude in degrees/degrees/meters (respectively), and the `beam dictionary`. In this case, beam_id == 0 is the UVBeam file hera.uvbeam, beam_id == 1 is an Airy disk with diameter 14 m, beam_id == 2 is a Gaussian beam with sigma 0.03, and beam_id == 3 is another Airy beam. When no shape parameter is written inline in the beam_dictionary (as with 3), pyuvsim will look for a default parameter below. So in this case, the beam_id == 3 ends up with a diameter of 12 m. The dictionary only needs to be as long as the number of unique beams used in the array, while the layout file specifies which antennas will use which beam type. This allows for a mixture of beams to be used, as in this example. Unassigned beams will be ignored (the given layout file does not use beams 2 or 3).
 
     Analytic beams may require additional parameters.
 
     - uniform = The same response in all directions. No additional parameters.
     - gaussian = Gaussian function shaped beam. Requires either an antenna diameter (in meters) or a standard deviation sigma (in radians). This standard deviation sets the width of the beam in zenith angle. Note that defining gaussian beams via `sigma` will be deprecated in the future.
```

Comment:
This may be out of scope for this PR, but does the impeding deprecation of sigma mean that we won't be able to specify non-chromatic gaussian beams in future? If so, we might want to rethink that.

Reply:
I've never been entirely clear on why we want to remove support for `sigma`. It seems like a perfectly sensible way of defining a power beam.
There is an option to give a power law evolution to the Gaussian beam width, but only if the width at a given frequency is specified by a `sigma`.